_config.yml

```diff
@@ -10,6 +10,10 @@ exclude:
   - alembic-jekyll-theme.gemspec
   - '.jekyll-cache'
 
+include:
+  - CNAME
+  - tietosuojaseloste.pdf
```

Comment:
Easier than `sudo cp` in a workflow file.

Reply:
Except that this doesn't get done on Netlify as the pdf doesn't exist yet.